webapp/src/modules/translation/locales/en.json

```
@@ -1412,7 +1412,11 @@
     "copy_link": "copy link",
     "share_on_twitter": "Share on twitter",
     "copied": "Copied",
-    "twitter_message": "I just shared a new List, Check it out: "
+    "twitter_message": {
```

The twitter message can be written in a single line using the newline character (`\n`).
```suggestion
    "message": "I just shared a new List 🫶,\nCheck it out:\n📍"
```